app/controllers/links_controller.rb

```diff
@@ -25,9 +25,18 @@ def external_url(type:, options: {})
       'https://projects.theforeman.org/projects/foreman/issues'
     when 'vmrc'
       'https://www.vmware.com/go/download-vmrc'
+    when 'docs'
+      params.require(:section)
+      guide, chapter, flavor = params.permit(:section, :chapter, :flavor).values_at(:section, :chapter, :flavor)
+      flavor ||= self.class.new_docs_flavor
+      docs_url(guide: guide, chapter: chapter, flavor: flavor)
     end
   end
 
+  def self.new_docs_flavor
+    Foreman::Plugin.find('katello').present? ? 'katello' : SETTINGS[:docs_os_flavor]
```

Comment:
I thought the canonical way of checking for plugins is with `Foreman::Plugin.installed?(plugin)`, although under the hood it does essentially the same thing you do here
```suggestion
    Foreman::Plugin.installed?('katello') ? 'katello' : SETTINGS[:docs_os_flavor]
```

Reply:
I copied this from `plugin_present?` and didn't check the API:
https://github.com/theforeman/foreman/blob/8dda6a6c4e7a0931e47ec81b4649215073afd43b/app/services/foreman/renderer/scope/macros/base.rb#L48-L56

I suppose that could also be replaced with `installed?` then.

Reply:
Ideally, but out of scope of this pr